infra/gcp-broad/main.tf

```diff
@@ -285,10 +277,6 @@ resource "google_container_node_pool" "vdc_nonpreemptible_pool" {
       enable_integrity_monitoring = true
       enable_secure_boot          = false
     }
-
-    workload_metadata_config {
-      mode = "GKE_METADATA"
-    }
```

Comment:
stacked on that other PR I presume, yeah?

Reply:
yup